File: classic/FizzBuzz/Lambda.rs
fn fizz_buzz(n: i32) -> String {
    let mut word = String::new();

    if n % 3 == 0 {
        word += "Fizz";
    }
    if n % 5 == 0 {
        word += "Buzz";
    }

    if word.is_empty() {
        word = n.to_string();
    }

    word
}

fn main() {
    (1..=100).for_each(|i| {
        println!("{}", fizz_buzz(i));
    });
}

File: classic/FizzBuzz/Lambda.py
def fizz_buzz(n: int) -> str:
    word = ""

    if n % 3 == 0:
        word += "Fizz"
    if n % 5 == 0:
        word += "Buzz"

    if len(word) == 0:
        word = str(n)

    return word


def main() -> None:
    for i in range(1, 101):
        print(fizz_buzz(i))


if __name__ == "__main__":
    main()
